File: CSharp/1218.cs
using System; 
using System.Linq;


class URI {

    static void Main(string[] args) 
    {   
        string[] lista;
        string a;
        int f=0;
        int m=0;
        int pares=0;
        int caso =1;
        

        while (true) 
        {
            a = Console.ReadLine();
            if(string.IsNullOrEmpty(a))
            {break;}
            if(caso>1)
            {Console.WriteLine();
}
            lista = Console.ReadLine().Split();

            for(int i=0;i<lista.Length;i+=2)
            {
                if(lista[i]==a)
                {
                    pares++;
                    if(lista[i+1]=="M")
                    {m++;}
                    else{f++;}
                }
            }
            Console.WriteLine($"Caso {caso}:");
            Console.WriteLine($"Pares Iguais: {pares}");
            Console.WriteLine($"F: {f}");
            Console.WriteLine($"M: {m}");
            m=0;
            f=0;
            pares=0;
            caso++;
        }
    }
}

File: Python/1218.py
caso = 1
while True:
    try:
        a = input().strip()
        if not a:
            break
        if caso > 1:
            print()

        lista = input().split()
        pares = 0
        m = 0
        f = 0

        for i in range(0, len(lista), 2):
            if lista[i] == a:
                pares += 1
                if lista[i + 1] == "M":
                    m += 1
                elif lista[i + 1] == "F":
                    f += 1

        print(f"Caso {caso}:")
        print(f"Pares Iguais: {pares}")
        print(f"F: {f}")
        print(f"M: {m}")
        caso += 1
        
    except EOFError:
        break
